pkg/v21/resource/clusterconfigmap/desired.go

```diff
@@ -30,6 +30,7 @@ func (r *StateGetter) GetDesiredState(ctx context.Context, obj interface{}) ([]*
 	values := map[string]string{
 		"baseDomain":   key.DNSZone(clusterConfig),
 		"clusterDNSIP": clusterDNSIP,
+		"clusterID":    key.ClusterID(clusterConfig),
```

Comment:
What about the label structure we have in `fmt`? Is this a thing?

Reply:
I don't think it needs to be namespaced like labels. As this is for use with helm templating.

I was thinking something like `.Values.cluster.id` or `.Values.giantswarm.cluster.id`.

Reply:
Other two keys (baseDomain and clusterDNSIP) are also "cluster" scoped, so IMO introducing cluster node would be redundant; I'd prefer if it's kept flat as originally proposed in the PR, at least for now, reconsider as more keys get added.